scripts/dependency-graph-generator/getDevDependencies.js

```diff
@@ -0,0 +1,32 @@
+const Project = require('@lerna/project');
+const PackageGraph = require('@lerna/package-graph');
+const path = require('path');
+const findGitRoot = require('../monorepo/index').findGitRoot;
+
+function flattenPackageGraph(rootPackages, projectGraph, packageList = []) {
+  rootPackages.forEach(packageName => {
+    packageList.push(packageName);
+
+    flattenPackageGraph([...projectGraph.get(packageName).localDependencies.keys()], projectGraph, packageList);
+  });
+
+  return packageList.sort().filter((v, i, a) => a.indexOf(v) === i);
+}
+
+/**
+ * Returns all the dev dependencies of a given package name
+ * @param {string} packageName including `@fluentui/` prefix
+ */
+async function getDevDependencies(packageName) {
```

Comment:
@ecraig12345  any util that exists for this currently ? I couldn't find any

Reply:
There might be something in `workspace-tools` but not all of those helpers have the most intuitive names. If you do add a new helper, I think it might make sense to put it under `scripts/monorepo` so it can be used by multiple things.

Reply:
Added to scripts/monorepo